aibolit/patterns/classic_setter/classic_setter.py

```diff
@@ -11,24 +8,38 @@ class ClassicSetter:
     def __init__(self):
```

Comment:
Remove constructor and add comment to class.

Reply:
ok